src/TransformationValidator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TransformationMigrate;
+
+use Keboola\TransformationMigrate\Exception\CheckConfigException;
+
+class TransformationValidator
+{
+    private array $config;
+
+    public function __construct(array $config)
+    {
+        $this->config = $config;
+    }
+
+    public function validate(): void
+    {
+        $this->validateBackend();
+        $this->validatePhases();
+    }
+
+    private function validateBackend(): void
+    {
+        $backends = array_map(
+            fn(array $v) => sprintf('%s-%s', $v['configuration']['backend'], $v['configuration']['type']),
+            $this->config['rows']
+        );
+        $uniqueBackends = array_unique($backends);
+
+        if (count($uniqueBackends) > 1) {
+            throw new CheckConfigException(sprintf(
+                'Transformations in the bucket "%s" don\'t have the same backend.',
```

Comment:
Navrhujem pridat `uniqueBackends` do vypisu, bude tam o uzitocnu informaciu viac.

Reply:
to vidíš na první pohled z detailu transformace, ale přidat to tam můžeme

Reply:
jj, islo mi iba o nejake davkove spustenie, nemusis to upravovat, ak netreba.